README.md

```diff
@@ -1,12 +1,11 @@
 # Amazon QLDB .NET Driver
 
-This is the .NET driver for [Amazon Quantum Ledger Database (QLDB)](https://aws.amazon.com/qldb/), which allows .NET developers
-to write software that makes use of AmazonQLDB.
-
 [![nuget](https://img.shields.io/nuget/v/Amazon.QLDB.Driver.svg)](https://www.nuget.org/packages/Amazon.QLDB.Driver/)
 [![license](https://img.shields.io/badge/license-Apache%202.0-blue)](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/LICENSE)
 [![AWS Provider](https://img.shields.io/badge/provider-AWS-orange?logo=amazon-aws&color=ff9900)](https://aws.amazon.com/qldb/)
 
+This is the .NET driver for [Amazon Quantum Ledger Database (QLDB)](https://aws.amazon.com/qldb/), which allows .NET developers
+to write software that makes use of Amazon QLDB.
 
```

Comment:
Go has "For getting started with the driver, see Go and Amazon QLDB." and a link to the getting started, which seems like it belongs here.

Reply:
The Getting started is missing the code snippet to list tables like Python and Node?

Reply:
The See Also section is missing items vs. the Python version?

Reply:
Added Dotnet Cookbook and DMV samples; but there's no equivalent for Driver Tutorial.

Reply:
@Dominitio I see the listing table snippet still isn't present?